File: fib.go
package main

import "fmt"

// Calculate the n^th Fibonacci number.
func fib(n int) int {
	if n < 2 {
		return n
	} else {
		return fib(n-1) + fib(n-2)
	}
}

func main() {
	res := fib(20)
	fmt.Println(res)
}

File: fib.java
// Calculate the n^th Fibonacci number.
public class fib {

  public static int fib(int n) {
    if (n < 2) {
      return n;
    } else {
      return fib(n - 1) + fib(n - 2);
    }
  }

  public static void main(String[] args) {
    int res = fib(20);
    System.out.println(res);
  }

}
